dbt/parser.py

```diff
@@ -256,6 +256,9 @@ def parse_schema_tests(tests, root_project, projects):
 
         for model_name, test_spec in test_yml.items():
             for test_type, configs in test_spec.get('constraints', {}).items():
```

Comment:
can we also check if `test_spec is None`? That would happen here:
```yml
test_model:
    constraints:
        # unique:
        #- some_field
```

Edge case, but would be good to avoid the ol stack trace :)